builders/build/templates/index.md

```diff
@@ -1,8 +1,32 @@
 ---
 title: Appchain Templates
-description: Learn more about the baseline appchain Templates (EVM and non-EVM) that support Tanssi out of the box to help you kickstart your appchain development.
-template: subsection-index-page.html
-hide:
- - toc
- - feedback
+description: Learn more about the baseline appchain Templates (EVM and non-EVM) that support Tanssi out of the box to help you kickstart your network development.
+icon: material-card-plus
```

Comment:
So I just reviewed the Learn index page PR and suggested to use this `:octicons-copy-24:` icon, so I think if that's the one we go with there, we should be consistent and also use it here. It will also help the icons give a more cohesive look